### PR TITLE
Travis: run `brew cask install`

### PR DESCRIPTION
**Pull request by commitay**
Refs: https://github.com/caskroom/homebrew-cask/issues/38528#issuecomment-329012321

Closes https://github.com/caskroom/homebrew-cask/issues/38528

### CODE REVIEW

File: ci/travis/script.sh [hunk 1]
```diff
@@ -21,6 +21,7 @@ if [[ ${#casks_wrong_dir[@]} -gt 0 ]]; then
 elif [[ ${#modified_casks[@]} -gt 0 ]]; then
   run brew cask _audit_modified_casks "${TRAVIS_COMMIT_RANGE}"
   run brew cask style "${modified_casks[@]}"
+  run brew cask install "${modified_casks[@]}"
```
Comment (reitermarkus) [inline ci/travis/script.sh]: This should also uninstall it afterwards.

Comment (commitay) [reply ci/travis/script.sh]: Why?

Comment (reitermarkus) [reply ci/travis/script.sh]: Why not? 😛

We have the Cask installed, so why not also test uninstalling it, while we're at it?

Comment (commitay) [reply ci/travis/script.sh]: `uninstall` (particually `launchctl`) and (I think) Applescript in general have issues when running headless. 

TBH I'm surprised I didn't encounter issues testing Casks that have a GUI progress bar `install` ([`netbeans`](https://travis-ci.org/caskroom/homebrew-cask/builds/274856289) and [`soapui`](https://travis-ci.org/caskroom/homebrew-cask/builds/274859761)). 

I'm not objecting to adding `uninstall` to the Travis test but I'd like to make sure we don't encounter any issues with testing `install` first.

**Review by reitermarkus — APPROVED** [review]
Let's see how this works out, and add `uninstall` later.

**Comment by commitay** [comment]
Just had a thought, we probably want to make `install` conditional so it doesn't run for mass updates.

**Comment by vitorgalvao** [comment]
> Applescript in general have issues when running headless.

We have (had?) an issue on that (by a user, not a maintainer).

**Review by miccal — APPROVED** [review]

**Comment by commitay** [comment]
This has been merged for a couple of days now and seems to be working fine.

A couple of observations.
- Caught an `install` error
    - https://travis-ci.org/caskroom/homebrew-cask/builds/276698027#L1632

-  if `depends_on macos` doesn't match the CI, Travis will fail 
    - https://travis-ci.org/caskroom/homebrew-cask/builds/276353352#L1625

- a couple of Casks are already installed on the CI images so `install` will skip them. (Currently installed Casks: `java`, `oclint`, `rubymotion`, `xquartz`)
    - CI [`brew cask list`](https://travis-ci.org/caskroom/homebrew-cask/builds/276850293#L1621)
    - https://travis-ci.org/caskroom/homebrew-cask/builds/276828773#L1629

- running `install` with `launchctl` (in a `postflight`) seems to work
    - [`softu2f`](https://github.com/caskroom/homebrew-cask/blob/3c77a6f94bada449aef75635fa3e441a279f8b60/Casks/softu2f.rb#L15-L26)
    - https://travis-ci.org/caskroom/homebrew-cask/builds/276850293#L1648

- `latest` Casks will download twice (`audit` and `install`.)

**Pull request merged**